source/Engine.cpp

```diff
@@ -683,7 +687,11 @@ void Engine::Step(bool isActive)
 			info.SetBar("target shields", target->Shields());
 			info.SetBar("target hull", target->Hull(), 20.);
 			info.SetBar("target disabled hull", min(target->Hull(), target->DisabledHull()), 20.);
-		
+			if(targetHit)
```

Comment:
Single-line loop or conditional bodies don't need braces per [style guide](http://endless-sky.github.io/styleguide/styleguide.xml). (Search for "single-line loop bodies".)